styles/banner.css

```diff
@@ -0,0 +1,68 @@
+.banner-container {
+    background-image: -moz-linear-gradient( 90deg, rgb(255,255,255) 0%, rgb(243,243,243) 79%, rgb(231,231,231) 100%);
+    background-image: -webkit-linear-gradient( 90deg, rgb(255,255,255) 0%, rgb(243,243,243) 79%, rgb(231,231,231) 100%);
+    background-image: -ms-linear-gradient( 90deg, rgb(255,255,255) 0%, rgb(243,243,243) 79%, rgb(231,231,231) 100%);
+    min-height: 400px;
+    width: 100%;
+    align-items: flex-end;
+    flex-direction: row;
+}
+
+.banner{
+    flex-direction: row;
+    align-items: flex-end;
+    flex-wrap: wrap;
+
+    margin: 52px 0 0 0;
+    box-sizing: border-box;
+    padding-right: 16px;
+}
+
+.banner__screen-img {
+    height: 348px;
+    width: 380px
+}  
+
+.banner__screen-img + .banner__info-container {
+    margin: 0 0 0 104px;
+}
+
+.banner__info-container {
+    padding-bottom: 77px;
+    max-width: 470px;
+}
+
+.banner__article {
+    margin: 0 0 35px 0;
+}
+
+.banner__subtitle, .banner__title {
+    color: #003d48;
+}
+
+.banner__subtitle {
+    font-weight: 300;
+    font-size: 30px;
+    word-spacing: 2.3px;
+    margin-bottom: 12px;
+}
+
+.banner__title {
+    text-transform: uppercase;
+    font-weight: 900;
+    font-size: 43px;
+    margin-bottom: 28px;
+}
+
+.banner__prh {
+    color:#929292;
+    font-size: 18px;
+}
+
+.banner__btn {
+    width: 129px;
```

Comment:
Instead of hard-coding width and height for buttons, consider defining paddings for the text inside.